scss/_popover.scss

```diff
@@ -153,8 +149,8 @@
 //
 // .popover-arrow is outer, .popover-arrow::after is inner
 
-.popover::before,
-.popover::after {
+.arrow::before,
```

Comment:
this should be moved  to `.popover` because this will overlaps with  `.arrow` from `.tooltip`